Ocean-back/Handlers/QuizQuestionHandlers/GetQuestionsHandler.cs

```diff
@@ -0,0 +1,44 @@
+﻿using Marten;
+using MediatR;
+using Ocean_back.Model;
+
+namespace Ocean_back.Handlers.QuizQuestionHandlers
+{
+    public class GetQuestions
+    {
+        public class Request : IRequest<Response>
+        {
+        }
+        public class Response
+        {
+            public class Item
+            {
+                public Guid CharId { get; set; }
+                public string Description { get; set; }
+                public List<Characteristic> Characteristics { get; set; }
+            }
+            public IEnumerable<Item> Items { get; set; } = Enumerable.Empty<Item>();
+        }
+        public class Handler : IRequestHandler<Request, Response>
+        {
+            private readonly IDocumentSession _session;
+            public Handler(IDocumentSession session) => _session = session;
+            public async Task<Response> Handle(Request request, CancellationToken cancellationToken)
+            {
+                var questions = await _session.Query<QuizQuestion>().ToListAsync();
+                if (!questions.Any())
+                    throw new Exception("No Questions available");
+
+                return new Response
+                {
+                    Items = questions.Select(questionc => new Response.Item
+                    {
+                        CharId = questionc.Id,
```

Comment:
QuestionId, and typo

Reply:
![kjamil_response](https://github.com/WebZaLeb/OCEAN-WebApp/assets/46653449/18232d7f-de57-4cd1-ad94-d8df53c7aae9)